### PR TITLE
Update AWS batch parameters and usage

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/tools!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config [hunk 1]
```diff
@@ -34,6 +34,11 @@ params {
   config_profile_contact = false
   config_profile_url = false
 
+  // AWSBatch
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: I don't think this should be in the template, but in the nf-core/configs instead

Comment (drpatelh) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: How do you mean? The parameters will need to be specified and therefore initialised at the pipeline level.

Comment (drpatelh) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: Maybe Im missing something about parameter propagation through configs 😅

Comment (MaxUlysse) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: I mean here: https://github.com/nf-core/configs/blob/master/conf/awsbatch.config

Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]:
```suggestion
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config [hunk 2]
```diff
@@ -34,6 +34,11 @@ params {
...
+  awsqueue = false
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]:
```suggestion
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config [hunk 3]
```diff
@@ -34,6 +34,11 @@ params {
...
+  awsregion = 'eu-west-1'
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]:
```suggestion
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config [hunk 4]
```diff
@@ -34,6 +34,11 @@ params {
...
+  awscli = '/home/ec2-user/miniconda/bin/aws'
```
Comment (MaxUlysse) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]:
```suggestion
```

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config (hunk 1 above)
Comment (drpatelh) [reply nf_core/pipeline-template/{{cookiecutter.name_noslash}}/nextflow.config]: Just remove that line?

**Comment by MaxUlysse** [comment]
cf https://github.com/nf-core/configs/pull/111

**Pull request merged**